easybuild/easyconfigs/g/GST-plugins-base/GST-plugins-base-1.9.1-foss-2016a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'GST-plugins-base'
+version = '1.9.1'
+
+homepage = 'http://gstreamer.freedesktop.org/'
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://gstreamer.freedesktop.org/src/gst-plugins-base']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+  ('GStreamer', '1.9.1'),
+  ('Automake', '1.15'),
+  ('Autoconf', '2.69'),
+  ('gettext', '0.19.7'),
+]
+
+builddependencies = [
+    ('XZ', '5.2.2', '-gettext-0.19.7'),
```

Comment:
I suspect this is intended to allow unpacking the sources, but this won't work due to https://github.com/hpcugent/easybuild-framework/issues/1376...